scripts/generate_conda_file.py

```diff
@@ -70,6 +70,7 @@
     "nvidia-ml-py3": "nvidia-ml-py3>=7.352.0",
     "papermill": "papermill>=0.15.0",
     "pydocumentdb": "pydocumentdb>=2.3.3",
+    "databricks-cli": "databricks-cli>=0.8.4"
```

Comment:
how confident are we that databricks-cli is a stable library with few breaking changes? in fastai, we had to change from  `>=` to `==` because of breaking changes

Reply:
no idea, so pinned.